### ISSUE TITLE
[Bug]: No pebbles on server

### ISSUE THREAD

**Issue by mercballs** [AllTheMods/All-the-mods-9-Sky#311]
### Possible Fixes

Yes

### Modpack Version

1.0.4

### What happened?

I am shift right clicking dirt/grass and get no pebbles. 1.04 running on a server. If I load a single player game I can get pebbles.

### latest.log

https://gist.github.com/mercballs/212a827ceb02d4f62d6ebf407890ac96

**Comment by wonni81** [comment]
Same issue.
Running ATM9 1.0.4 server on Unraid.

**Comment by mercballs** [comment]
I found that the pack installed fine, but some of the server files were not unzipped correctly into the folder. For some reason the contents of the "kubejs" folder were just put into the root of the server folder. The server will start, but all of the kubejs scripts do not load.

I was able to fix it by extracting the "kubejs" folder and its proper structure to the server folder. It seems that one of the install scripts is screwed up. Not sure if this on the unraid side or what.
